build.sbt

```diff
@@ -45,7 +45,8 @@ lazy val examples = project
   .dependsOn(core.jvm)
   .settings(
     name                 := "shellfish-examples",
-    Compile / run / fork := true
+    Compile / run / fork := true,
+    run / connectInput   := true
```

Comment:
Are you sure these are not required just on your setup?

Reply:
In Linux yes, I was getting a EOF error if I didn't placed that sadly :/ haven't tried in Windows tho

Reply:
That's the precise reason I'm asking, I'm getting and EOF too, so I don't think this is a general solution